test/ClangImporter/macros.swift

```diff
@@ -109,6 +109,20 @@ func testBitwiseOps() {
   _ = (BIT_SHIFT_1 | BIT_SHIFT_2) as CInt
   _ = BIT_SHIFT_3 as CLongLong
   _ = BIT_SHIFT_4 as CUnsignedInt
+
+  var attributes = 0 as CInt
+  attributes |= ATTR_BOLD
+  attributes |= ATTR_ITALIC
+  attributes |= ATTR_UNDERLINE
+}
+
+func testAddSubtract() {
+  _ = ADD_ZERO
+  _ = ADD_ONE
+  _ = ADD_TWO
+
+  _ = SUB_ONE
+  _ = SUB_ZERO
```

Comment:
Hm. I'd really like a test to make sure the *values* are right too. I was hoping we had a SILGen test for this already but I don't see one. The closest I can see is ctypes_ir.swift, which checks for the values of imported enums.

Are you comfortable writing a test with -emit-silgen, or would you prefer I add one on your behalf after this gets merged?